### PR TITLE
Default to public interface for os-capacity

### PR DESCRIPTION
**Pull request by assumptionsandg**

### CODE REVIEW

**Review by markgoddard — DISMISSED** [review]

**Review by markgoddard — COMMENTED** [review]
Any docs update required?

File: doc/source/configuration/monitoring.rst [hunk 1]
```diff
@@ -155,12 +155,14 @@ the OpenStack application credentials in ``kayobe/secrets.yml`` as:
 
 The Keystone authentication URL and OpenStack region can be changed
 from their defaults in ``stackhpc-monitoring.yml`` should you need to
-set a different OpenStack region for your cloud. The authentication
-URL is set to use ``kolla_internal_fqdn`` by default:
+set a different OpenStack region for your cloud or change the default
+interface. The authentication URL is set to use ``kolla_internal_fqdn``
```
Comment (markgoddard) [inline doc/source/configuration/monitoring.rst]: external

**Review by markgoddard — DISMISSED** [review]

File: doc/source/configuration/monitoring.rst [hunk 2]
```diff
@@ -158,6 +158,7 @@ project domain name in ``stackhpc-monitoring.yml``:
     stackhpc_os_capacity_project_name: <openstack_project_name>
     stackhpc_os_capacity_domain_name: <openstack_project_domain_name>
     stackhpc_os_capacity_openstack_region_name: <openstack_region_name>
+    stackhpc_os_capacity_openstack_interface: <openstack_interface>
 
 Additionally, you should ensure these credentials have the correct permissions
 for the exporter. If you are deploying in a cloud with internal TLS, you may be required
```
Comment (markgoddard) [inline doc/source/configuration/monitoring.rst]: Actually internal TLS comment is less relevant now

File: doc/source/configuration/monitoring.rst [hunk 3]
```diff
@@ -158,15 +158,7 @@ project domain name in ``stackhpc-monitoring.yml``:
     stackhpc_os_capacity_project_name: <openstack_project_name>
     stackhpc_os_capacity_domain_name: <openstack_project_domain_name>
     stackhpc_os_capacity_openstack_region_name: <openstack_region_name>
```
Comment (markgoddard) [inline doc/source/configuration/monitoring.rst:160]: It needs to be in this list if there is no default

**Comment by markgoddard** [comment]
Bump

**Comment by Alex-Welsh** [comment]
@assumptionsandg is this PR still relevant? Please either update it or close it

**Pull request closed without merging**